app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,31 @@
 class PostsController < ApplicationController
+  include PostsHelper
+
   def index
     @user = User.find_by_id(params[:user_id])
     @posts = @user.posts
   end
 
+  def new
+    @user = User.find_by_id(params[:user_id])
+  end
+
+  def create
+    @user = User.find_by_id(params[:user_id])
+    @post = Post.new
```

Comment:
- [ ] Please your post form is not functioning so I couldn't create a post so please make sure your fix this issue and ensure that creating and liking posts works perfectly :+1: 
  
  [Screencast from 06-14-2023 11:23:13 PM.webm](https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/92303891/92d5d9d7-fd3f-425a-9d3f-fca3219ebc8c)

Reply:
As you can see for me the create comment and create post are working @Owusu-Desmond 
https://github.com/Alejandro-Bernal-M/rails-blog-app/assets/118059775/ed70a85b-c117-4156-8398-6cf60578895f